server/aws-lsp-codewhisperer/src/language-server/codeWhispererServer.ts

```diff
@@ -573,6 +605,28 @@ export const CodewhispererServerFactory =
 
         logging.log('Amazon Q Inline Suggestion server has been initialised')
 
+        const logSupplementalContext = (supplementalContext: CodeWhispererSupplementalContext | undefined) => {
```

Comment:
Isn't this too verbose, i see in vscode toolkit codebase they use debug logging to log this information whereas it seems we log always

Reply:
hm, yeah you're right. I added it as normal log since we don't have different log levels implemented.

Reply:
Removed excessive logging in last commit